pages/gigabit/portfolio.json

```diff
@@ -0,0 +1,352 @@
+    "Lead Organization": "Bonner Springs Fire Department",
+    "Other Partner Organizations": "McGill University",
+    "Photo": "",
+    "Link": null,
+    "Tech Tags": "AR",
+    "Issue Area Tags": "Open Innovation"
+  },
+  {
+    "Year": 2014,
+    "City": ["KC"],
+    "Project": "Gigabit Cafe",
+    "Project Summary": "This project piloted the first public test of the Software Lending Library in Kansas City. The project aims to bring the benefits of gigabit internet connectivity to clients and neighbors in an underserved area of Kansas City. More than just a way to borrow free programs, the Software Lending Library was created to enhance digital inclusion by enabling people to access software needed to be successful in a technology-driven world.",
+    "Outcomes": "This project was demoed at the 2014 US Ignite Summit as well as GIGTANK Demo Day in Chattanooga.",
+    "Gigabit Advantage": "Using the ultra-high speed connection to deliver the applications allows users to do homework, perform routine tasks and generate creative output from typically low-performing or older computers and devices. The Lending Library has tremendous potential to mediate the effects of unequal access to productivity tools in the community.",
+    "Lead Organization": "Roncilliation Services",
+    "Other Partner Organizations": "Kansas Public Library",
+    "Photo": "/img/pages/gigabit/logos/gigabitcafe.jpg",
+    "Link": "http://softwarelendinglibrary.org/",
+    "Tech Tags": "",
+    "Issue Area Tags": "Open Innovation, Digital Inclusion, Web Literacy"
+  },
+  {
+    "Year": 2014,
+    "City": ["KC"],
+    "Project": "Gigabots",
+    "Project Summary": "The Gigabots bring connected devices to classrooms using existing educational robotics platforms. A Gigabot is a combination of hardware, software and a cloud platform that connects Gigabots to each other from any location in real-time.",
+    "Outcomes": "Students participating in the pilot learned basic JavaScript, how to program the Gigabots remotely, and how to implement the touch sensor into programming robot behaviors. This project was demoed at MakerFaire KC, at SummerQuest 2014 in the Bronx, as well as at the 2015 and 2016 US Ignite Summits.",
+    "Gigabit Advantage": "Gigabots expand traditional robotics-league curriculum with IoT and programming.  With sensor-based interactions, high speed and low latency are critical to the user experience. “Gigabots Live” offers a low cost, low latency video streaming device suitable for use with internet connected robotics devices.",
+    "Lead Organization": "Big Bang",
+    "Other Partner Organizations": "Staley High School",
+    "Photo": "/img/pages/gigabit/logos/14715602331_fe3448fd97_o.jpg",
+    "Link": "https://getbigbang.com/",
+    "Tech Tags": "Robotics, IoT",
```

Comment:
I guess we could...currently this isn't used by anything actually. I figured we'd array-ify it once we make it into a real service.